tests/functional/cylc-ping/04-check-keys-remote.t

```diff
@@ -37,19 +37,22 @@ __FLOW_CONFIG__
 run_ok "${TEST_NAME_BASE}-validate" cylc validate "${SUITE_NAME}" \
     -s "CYLC_TEST_PLATFORM='${CYLC_TEST_PLATFORM}'"
 suite_run_ok "${TEST_NAME_BASE}-run" cylc run "${SUITE_NAME}" \
-    -s "CYLC_TEST_PLATFORM='${CYLC_TEST_PLATFORM}'"
+    --debug -s "CYLC_TEST_PLATFORM='${CYLC_TEST_PLATFORM}'"
 RRUND="cylc-run/${SUITE_NAME}"
 RSRVD="${RRUND}/.service"
 poll_grep_suite_log 'Holding all waiting or queued tasks now'
 SSH='ssh -n -oBatchMode=yes -oConnectTimeout=5'
+ 
 ${SSH} "${CYLC_TEST_HOST}" \
 find "${RSRVD}" -type f -name "*key*"|awk -F/ '{print $NF}'|sort >'find.out'
 cmp_ok 'find.out' <<__OUT__
+client_${CYLC_TEST_INSTALL_TARGET}.key
 client.key_secret
-client_$CYLC_TEST_PLATFORM.key
```

Comment:
Need to go back to the original order. I guess `.` comes before `_` alphabetically

Reply:
Ah, changing order will break the test locally. Needs the sort.

Reply:
This order passes both on GH Actions and locally (can't imagine `sort` works differently on RHEL and Ubuntu):
```
cmp_ok 'find.out' <<__OUT__
client.key_secret
client_${CYLC_TEST_INSTALL_TARGET}.key
server.key
__OUT__
```
Anyway, whatever works.

Reply:
Interesting that you are getting that passing locally...
![image](https://user-images.githubusercontent.com/37735232/106266055-d9a66100-621f-11eb-8339-43bc276397a5.png)

Reply:
Well maybe `sort` behaves differently on RHEL and RHEL 😢

Reply:
Perhaps consider `sort -d` if behaviour is inconsistent.

Reply:
IDE virtual terminals don't can exhibit some very strange behaviour, I remember fighting some strange buffering behaviour in eclipse and pycharm before I moved to using regular terminals (but hey, learned a bit about text buffering in the process 🤷).

Here are two other recent cases of test failures in cylc-flow caused by IDE terminals:

* https://github.com/cylc/cylc-flow/pull/3989
* https://github.com/cylc/cylc-flow/issues/3233